.github/workflows/utils/sql-proxy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -ex
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shared Cloud SQL Proxy setup
+# Presumes the following variables in ci-setup.json:
```

Comment:
Should we add explicit validations to exit with an error message if these variables are not set?

Reply:
This is where I need to confirm that CLOUD_SQL_CONNECTION_NAME is the variable used in all samples that will use this helper, but yes, if that is not set, then probably fail out. UNIX_SOCKET_DIR is a new variable that needs to be set to a specific value in tests, but should by default be `/cloudsql` in the actual sample itself, so I'll confirm this before forcing this to be set.